code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -100,10 +100,10 @@
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guninhands_left.dmi', "right_hand" = 'icons/mob/in-hand/right/guninhands_right.dmi')
 	origin_tech = "combat=5;materials=2"
 	w_class = W_CLASS_MEDIUM
-	max_shells = 20
-	burst_count = 4
+	max_shells = 16
```

Comment:
Personally I think the capacity should be evenly divisible by its burst count, so you aren't left with a half-burst of leftover bullets at the end.